### PR TITLE
CMS-814: Implement the new design of the main table view

### PR DESCRIPTION
**Pull request by Flight**
This PR implement the redesign of the main filer view with fully working actions and buttons, except remove file button (need proper URL).

### CODE REVIEW

**Comment by vxsx** [comment]
codewise looks good, although i don't fully understand the "temporary" stuff

**Comment by Flight** [comment]
@vxsx it will be reimplemented from the BE side, so I just emulating the old stuff for the new buttons/dropdowns.

**Comment by vxsx** [comment]
ah, i see
:+1:

**Comment by FinalAngel** [comment]
@Flight we are also in the habit of attaching screenshots to show whats going. This helps a lot :)

File: filer/static/filer/js/base.js [hunk 1]
```diff
@@ -21,5 +20,36 @@ var Cl = window.Cl || {};
         if (Cl.Toggler) {
             new Cl.Toggler();
         }
+
+        $(document).on('click', '.dropdown-menu-checkboxes', function (clickEvent) {
+            clickEvent.stopPropagation();
+        });
+
+        // Temporary stuff here, please do not review
```
Comment (FinalAngel) [inline filer/static/filer/js/base.js]: please write appropriate comments like ``mocking the action buttons to work in frontend UI``

File: filer/templates/admin/filer/folder/directory_listing.html [hunk 2]
```diff
@@ -96,55 +96,36 @@ <h1 class="folder_header"><a class="backtoroot" href="{% url 'admin:filer-direct
...
+                            <span class="icon fa fa-folder"></span>
+                            {% trans "New Folder" %}
+                        </a>
+                    {% endif %}
+                    {% if permissions.has_add_children_permission and not folder.is_root %}
+                        <a href="#" id="id_upload_button" title="{% trans 'Upload Files' %}" class="navigator-button navigator-button-upload">
+                            <span class="icon fa fa-file"></span>
+                            {% trans "Upload Files" %}
+                        </a>
+                        {% include 'admin/filer/tools/upload_button_js.html' %}
```
Comment (FinalAngel) [inline filer/templates/admin/filer/folder/directory_listing.html]: double quotes

File: filer/templates/admin/filer/folder/directory_table.html [hunk 3]
```diff
@@ -56,22 +56,24 @@
                                 {% endif %}
                             </td>
                             <td class="column-action">
-                                <a href="#" class="fa fa-pencil"></a>
-                                <a href="#" class="fa fa-trash-o"></a>
+                                {% if subfolder.file_type == 'Folder' %}
```
Comment (FinalAngel) [inline filer/templates/admin/filer/folder/directory_table.html]: double quotes

**Pull request merged**